Makefile

```diff
@@ -297,63 +297,71 @@ ifeq "$(FLEXDLL_SUBMODULE_PRESENT)" ""
 else
   BOOT_FLEXLINK_CMD = FLEXLINK_CMD="../boot/ocamlrun ../flexdll/flexlink.exe"
   CAMLOPT := OCAML_FLEXLINK="boot/ocamlrun flexdll/flexlink.exe" $(CAMLOPT)
-  FLEXDLL_DIR=$(if $(wildcard flexdll/flexdll_*.$(O)),"+flexdll")
+  FLEXDLL_DIR=$(if $(wildcard flexdll/flexdll_*.$(O)),+flexdll)
```

Comment:
Travis complains about this change. The value of FLEXDLL_DIR must be a string in OCaml syntax, so the quotes are needed.

Reply:
It's not this - is a mistake in one of the macros further down. I'll push a fix in a second